Controls.UserDialogs.Maui/Platforms/iOS/UserDialogsImplementation.cs

```diff
@@ -40,8 +40,8 @@ public partial class UserDialogsImplementation
                 FontFamily = config.MessageFontFamily,
                 Position = config.Position.ToNative(),
             };
-            bar.BackgroundColor ??= config.BackgroundColor.ToPlatform();
-            bar.MessageColor ??= config.MessageColor.ToPlatform();
+            bar.BackgroundColor = config.BackgroundColor.ToPlatform();
+            bar.MessageColor = config.MessageColor.ToPlatform();
```

Comment:
I suppose this should be done like for snackbar
```
bar.BackgroundColor = config.BackgroundColor?.ToPlatform() ?? bar.BackgroundColor;
bar.MessageColor = config.MessageColor?.ToPlatform() ?? bar.MessageColor;
```

Also, fix this for mac please